cypress/e2e/model-card.cy.ts

```diff
@@ -0,0 +1,78 @@
+describe('Model with model card only', () => {
+  before(() => {
+    cy.log('Navigate to Upload page and json tab')
+    cy.visit('/upload')
+    cy.get('[data-test=uploadJsonTab]').click({ force: true })
+
+    cy.log('Selecting schema and inputting metadata')
+    cy.get('[data-test=selectSchemaInput]').trigger('mousedown', { force: true, button: 0 })
+    cy.fixture('schema_names.json').then((schemaNames) => {
+      cy.get(`[role=option]:contains(${schemaNames.model})`).click()
+    })
+
+    cy.fixture('minimal_metadata_modelcard.json').then((metadata) => {
+      metadata.buildOptions.uploadType = 'Model card only'
+      cy.get('[data-test=metadataTextarea]')
+        .clear()
+        .type(JSON.stringify(metadata), { parseSpecialCharSequences: false, delay: 0 })
+    })
+
+    cy.log('Submitting model')
+    cy.get('[data-test=warningCheckbox]').click()
+    cy.get('[data-test=submitButton]').click()
+    cy.url().should('contain', '/model/', { timeout: 10000 })
```

Comment:
The timeout should be on the command, not the assertion:
`cy.url({ timeout: 10000 }).should('contain', '/model/')`